samples/tab-product-inspection/nodejs/Manifest/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "<<manifest-Id>>",
+  "packageName": "com.teams.productinspection",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Product Inspection",
+    "full": "Product Inspection sample"
+  },
+  "description": {
+    "short": "Product Inspection",
+    "full": "This app allows users to scan a product, capture a image and mark it as approved/rejected."
+  },
+  "accentColor": "#5558AF",
+  "staticTabs": [
+    {
+        "entityId": "index",
```

Comment:
Why the entity id is "index"??
This tab is related to product inspection so you should give meaningful name.